docs/user/reference/environment-variables.rst

```diff
@@ -51,6 +51,19 @@ All :doc:`build processes </builds>` have the following environment variables au
 
     :Example: ``/home/docs/checkouts/readthedocs.org/user_builds/project/envs/version``
 
+.. envvar:: READTHEDOCS_OUTPUT
+
+    Base path for output formats when using custom :doc:`build.commands </build-customization>` to generate output.
+    You need to concatenate an output format,
+    for instance ``$READTHEDOCS_OUTPUT/html/`` or ``$READTHEDOCS_OUTPUT/pdf/``.
+    You also need to create the directory,
+    for instance by adding a command ``mkdir -p $READTHEDOCS_OUTPUT/html/`` before moving outputs into the destination.
```

Comment:
This is a great addition! Yes, this belongs in this reference.

It also reminds me that we discussed to have `READTHEDOCS_OUTPUT_HTML` and `READTHEDOCS_OUTPUT_PDF` etc? That would make reference writing a bit simpler, even though there would be more variables to document.

For instance, if/when we support multiple PDFs, this might have something extra to say about file name conventions (I'm not sure how it's gonna work, just as an example).